alacritty/src/event.rs

```diff
@@ -227,23 +224,30 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
     }
 
     fn change_font_size(&mut self, delta: f32) {
-        self.resize_pending.font_size = Some(FontResize::Delta(delta));
+        *self.font_size = max(*self.font_size + delta, Size::new(FONT_SIZE_STEP));
+        self.display_update_pending.font =
+            Some(self.config.font.clone().with_size(*self.font_size));
```

Comment:
I'm not sure if I've mentioned this in the previous comment, but since this line is longer than 100 columns, I think it makes sense to extract the stuff inside the `Some` into a `let font = self.config...;`.

If we have two lines anyways, it's usually a bit more readable to have separate statements.